package.json

```diff
@@ -55,12 +55,12 @@
   },
   "devDependencies": {
     "@types/bootstrap": "^5.0.17",
+    "@types/dropzone": "^5.7.4",
     "@types/fancybox": "^3.5.2",
     "@types/file-saver": "^2.0.1",
     "@types/jquery": "^3.5.1",
     "@types/jqueryui": "^1.12.14",
     "@types/luxon": "^2.0.0",
-    "@types/tinymce": "^4.6.0",
```

Comment:
why were these removed?

Reply:
* We use version ^5.10 and this version comes with type definitions (https://www.tiny.cloud/docs/changelog/#550-2020-09-29)
* [@types/tinymce are made for TinyMCE version 4.6](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/tinymce/index.d.ts#L1)